medusa/clients/torrent/generic.py

```diff
@@ -301,27 +252,16 @@ def test_authentication(self):
         :return:
         :rtype: tuple(bool, str)
         """
-        try:
```

Comment:
have you tested the test_authentication method with other clients? Like with deluge, transmission, rtorrent, etc?

Reply:
Only with transmission, but there is nothing changed for them, so I do not expect any problems.
I dont' have all those clients available.

Reply:
I'll see if I can run some tests. I still have some of the docker containers from the download_handler pr.